exercises/chapter4/test/Main.purs

```diff
@@ -70,8 +70,10 @@ Note to reader: Delete this line to expand comment block -}
       test "Exercise - isPrime" do
```

Comment:
Should we test negatives too?

Reply:
They are in the function domain so I think it's reasonable to test them.